src/citrine/resources/project_member.py

```diff
@@ -0,0 +1,18 @@
+from citrine.resources.project_roles import ROLES
+from citrine.resources.user import User
+
+
+class ProjectMember:
+    """A Member of a Project."""
+
+    def __init__(self,
+                 user: User,
+                 project: 'Project',
+                 role: ROLES):
+        self.user: User = user
+        self.project: 'Project' = project
```

Comment:
What hath @dannycodes wrought?
You mentioned this in person, but what does this string typing do?

Reply:
https://www.python.org/dev/peps/pep-0484/#forward-references

Reply:
Lol the inline comment is clearly not where I thought it was

Reply:
Wow.

Reply:
added a comment